### PR TITLE
`s` commands: move `help_msg_*[]` arrays to top

### PR DESCRIPTION
**Pull request by MaskRay**

### CODE REVIEW

File: libr/core/cmd.c [hunk 1]
```diff
@@ -3148,6 +3148,7 @@ R_API void r_core_cmd_init(RCore *core) {
 	r_cmd_add (core->rcmd, "info",     "get file info", &cmd_info);
 	r_cmd_add (core->rcmd, "cmp",      "compare memory", &cmd_cmp);
 	r_cmd_add (core->rcmd, "seek",     "seek to an offset", &cmd_seek);
```
Comment (radare) [inline libr/core/cmd.c]: Maybe we should have a struct defining all commands, descripttion, callbacks and constructors and just do a loop

File: libr/core/cmd_seek.c [hunk 2]
```diff
@@ -7,6 +7,65 @@
...
+};
+
+static const char *help_msg_ss[] = {
+	"Usage: ss", "", " # Seek silently (not recorded in the seek history)",
+	"s?", "", "Works with all s subcommands",
+	NULL
+};
+
+static void cmd_seek_init(void) {
+	DEFINE_CMD_DESCRIPTOR(s);
```
Comment (radare) [inline libr/core/cmd_seek.c]: Missing spaces

File: libr/core/cmd.c (hunk 1 above)
Comment (MaskRay) [inline libr/core/cmd.c]: Yes, it should be and it is a certainly a step in the migration... Currently, can I add these spaghetti `cmd_*_init` mixed in the code so as to have no merge conflicts?

File: libr/core/cmd_seek.c (hunk 2 above)
Comment (MaskRay) [inline libr/core/cmd_seek.c]: Done.

File: libr/core/cmd.c (hunk 1 above)
Comment (radare) [inline libr/core/cmd.c]: you could do a single pr instead of 9000 to avoid merge conflicts :P

**Pull request merged**